Source/bmalloc/libpas/src/libpas/pas_thread.c

```diff
@@ -0,0 +1,206 @@
+/*
```

Comment:
Also, in general, please follow to libpas / WebKit coding style. Like, we should not put `*` to the variable side for pointers.